src/util/http/websocket/WebSocketSession.cpp

```diff
@@ -76,9 +96,14 @@ net::awaitable<void> WebSocketSession::acceptAndWait(
     co_await (waitForServerEvents() && handleClientCommands());
 
   } catch (boost::system::system_error& error) {
+    if (cancelOnClose_) {
+      tryToCancelQuery();
+    }
+    static const std::array<boost::system::error_code, 4> allowedCodes{
```

Comment:
I have to use the explicit type because the different error codes all have different types that are type erased to `boost::system:error_code`. This is also the reason why this can't be constexpr.